src/main/java/edu/njit/jerse/automation/AsheAutomation.java

```diff
@@ -188,26 +192,41 @@ private static String fullyQualifiedMethodReference(String packageAndClassName,
      *                     files being processed.
      *                     Example: /absolute/path/to/project
      *                 </li>
+     *                 <li>
+     *                     LLM argument - "gpt-4" or "manual"
+     *                     - gpt-4 will run the GPT-4 model
+     *                     - manual will run the manual response the user provides in predefined_responses.txt
+     *                 </li>
      *             </ol>
      * @throws IOException if an I/O error occurs while accessing the directory
      */
     public static void main(String[] args)
             throws IOException {
-        if (args.length != 2) {
-            LOGGER.error("Invalid number of arguments. Expected 2 arguments: <Directory Path> <Project Root Path>");
+        if (args.length != 3) {
+            LOGGER.error("Invalid number of arguments. Expected 3 arguments: <Directory Path> <Project Root Path> <LLM>");
             System.exit(1);
         }
 
         String directoryPath = args[0];
         String projectRootPath = args[1];
 
+        // LLM argument - either gpt-4 or manual (for now)
+        String model = args[2];
+
+        // TODO: Add more models here.
+        // Example: Arrays.asList("llama", "palm", "grok");
+        Set<String> models = new HashSet<>(Arrays.asList("gpt-4", "manual"));
```

Comment:
This list also appears at `Ashe.java` line 169. Please consolidate these so that adding a new model only requires changing the list accepted by the argument once (maybe write a shared static method to check this?).